controllers/endpoints.controller.js

```diff
@@ -0,0 +1,5 @@
+const endpoints = require('../endpoints.json');
+
+exports.getEndpoints = (req, res) => {
+  res.status(200).send({ endpoints });
```

Comment:
To keep your responses consistent across the project, you could parse this JSON object before sending it in your response